app/2.1.x/clustering.md

```diff
@@ -78,6 +78,23 @@ This makes Kong clusters **eventually consistent**.
 
 [Back to top](#introduction)
 
+### Use read-only replicas when deploying Kong clusters with Postgres
+
+When using Postgres as the backend storage, you can optionally enable
+Kong to serve read queries from a separate database instance.
+
+One of the common use case of this feature is to deploy Kong with the
+Amazon Aurora service as backend storage. As Aurora natively supports
+read only instances, enable the read-only connection support in Kong
+greatly reduces the load on the main database instance as read-only
+queries are no longer sent to it.
+
+To learn more about how to configure this feature, refer to the
+<a href="/{{page.kong_version}}/configuration/#datastore-section">Datastore section</a>
```

Comment:
```suggestion
[Datastore section](/enterprise/{{page.kong_version}}/property-reference/#datastore-section)
```

Reply:
This was changed in 09e979596d19d1497326cf504fab773f345ea136 to refer to the community version link in the community doc. Enterprise doc link will be changed accordingly.